server/plugin/hooks.go

```diff
@@ -47,14 +54,57 @@ func (p *Plugin) MessageHasBeenPosted(c *plugin.Context, post *model.Post) {
 		return
 	}
 
+	if len(post.FileIds) > 1 {
+		p.logAndSendErrorToUser(mattermostUserID, channel.Id, "Cannot send more than one file attachment at a time.")
+		return
+	}
+
 	token, err := p.ParseAuthToken(user.OAuth2Token)
 	if err != nil {
-		p.logAndSendErrorToUser(mattermostUserID, channel.Id, fmt.Sprintf("error occurred while decrypting token. Error: %s", err.Error()))
+		p.logAndSendErrorToUser(mattermostUserID, channel.Id, fmt.Sprintf("Error occurred while decrypting token. Error: %s", err.Error()))
```

Comment:
The error that is sent to the user should not be this. It should be generic like "Something went wrong." and the real error should be logged in the server logs.

Reply:
The function `logAndSendErrorToUser` does this only. It logs the error passed to it and send a generic message to the user